conferences/2018/javascript.json

```diff
@@ -615,5 +615,15 @@
     "city": "Kilkenny",
     "country": "Ireland",
     "twitter": "@NodeConfEU"
-  }
+  },
+  
+  {
+    "name": "AngularConnect",
+    "url": "http://angularconnect.com/",
+    "startDate": "2018-11-06",
+    "endDate": "2018-11-07",
+    "city": "London",
+    "country": "U.K.",
+    "twitter": "@AngularConnect"
+  },
```

Comment:
Can you remove this extra coma? JSON will fail otherwise.

Reply:
Done